### PR TITLE
add pipe to bash script

### PR DESCRIPTION
**Pull request by nimakaviani**
also updates the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,19 @@ A container engine is needed locally such as:
...
-## Quickstart
+## Getting Started
 
 If you are interested in running idpbuilder in Codespaces through your browser, check out the [Codespaces](#running-in-codespaces) section.
 
-### Download and install the idpbuilder
+### Quick Install
 
-Download the latest release with the commands:
+You can *carefully* run the following script to install `idpbuilder`
```
Comment (blakeromano) [inline README.md:39]: I am not sure why adding `*carefully*` here is adding. How are we advocating that folks carefully run the script?

Comment (omrishiv) [reply README.md:39]: I think the intent is to make people aware that running this type of command *could* be dangerous if they don't review the script. Maybe: 

We offer this command as a convenience for installing idpbuilder (be sure to check the script first if you are concerned):

Comment (nimakaviani) [reply README.md:39]: good call. will update the info there.

File: README.md [hunk 2]
```diff
@@ -30,13 +30,18 @@ A container engine is needed locally such as:
...
-### Download and install the idpbuilder
+### Quick Install
 
-Download the latest release with the commands:
+The following command can be used as a convenience for installing `idpbuilder`, (be sure to check the script first if you are concerned):
+```
+bash <(curl -fsSL https://raw.githubusercontent.com/cnoe-io/idpbuilder/main/hack/install.sh)
+```
+
+or download the latest release with the follwoing commands:
```
Comment (blakeromano) [inline README.md:44]: typo with `follwoing` -> `following

**Review by blakeromano — APPROVED** [review]
Outside of typo 🚀

File: README.md [hunk 3]
```diff
@@ -30,13 +30,18 @@ A container engine is needed locally such as:
...
 
 If you are interested in running idpbuilder in Codespaces through your browser, check out the [Codespaces](#running-in-codespaces) section.
 
-### Download and install the idpbuilder
+### Quick Install
 
-Download the latest release with the commands:
+The following command can be used as a convenience for installing `idpbuilder`, (be sure to check the script first if you are concerned):
+```
+bash <(curl -fsSL https://raw.githubusercontent.com/cnoe-io/idpbuilder/main/hack/install.sh)
```
Comment (omrishiv) [inline README.md:41]: nit: the convention is usually: 

```
curl -fsSL https://raw.githubusercontent.com/cnoe-io/idpbuilder/main/hack/install.sh | bash
```

**Review by omrishiv — COMMENTED** [review]
Small nit, but LGTM

File: README.md (hunk 3 above)
Comment (nabuskey) [reply README.md:41]: Second this. Easier to read imo.

Comment (nimakaviani) [reply README.md]: fixed

**Review by nabuskey — APPROVED** [review]
LGTM

**Pull request merged**